### PR TITLE
Filledstars

### PR DESCRIPTION
**Pull request by Aaron-Fink**
added log click to clickable items and made cursor a pointer on all clickable items. fixed tests that were failing.

### CODE REVIEW

File: client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx [hunk 1]
```diff
@@ -12,24 +12,6 @@ const ImageGalleryImageSlide = ({
   thumbSplitArr, thumbDisplayArr, setThumbDisplayArr, setOverviewModal, LogClick,
 }) => (
   <div id="imagegallery-mainimage-container">
```
Comment (greta-schock) [inline client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx:14]: What happened here

Comment (Aaron-Fink) [reply client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx:14]: the line above 11?

**Review by greta-schock — APPROVED** [review]
Good looks

File: client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx (hunk 1 above)
Comment (greta-schock) [reply client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx:14]: There were so many components in the container and then it's all gone and I was curious! But ultimately approved!

**Pull request merged**